### PR TITLE
raylib: Init at 2.6.0

### PR DESCRIPTION
**Pull request by svantepolk**
###### Motivation for this change
I like raylib and would like it to be more accessible (since that's kind of the point of raylib).

###### Things done
- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/libraylib/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  inherit name;
+
+  meta = {
+    description = "A simple and easy-to-use library to enjoy videogames programming";
+    repo = "raylib";
+    owner = "raysan5";
+    license = stdenv.lib.licenses.zlib;
+    maintainers = [ stdenv.lib.mantainers.svantepolk ];
+    rev = version;
+    sha256 = hash;
```
Comment (ryneeverett) [inline pkgs/development/libraries/libraylib/default.nix]: I'm not familiar with some of these [meta-attributes](https://nixos.org/nixpkgs/manual/#chap-meta). Am I missing something?

Comment (svantepolk) [reply pkgs/development/libraries/libraylib/default.nix]: Mistakenly merged from a prior git source configuration I was trying. I've addressed this in a fixup.

File: pkgs/development/libraries/raylib/default.nix [hunk 2]
```diff
@@ -0,0 +1,30 @@
+with import <nixpkgs> { };
```
Comment (doronbehar) [inline pkgs/development/libraries/raylib/default.nix:1]: @svantepolk Could you please use:

```suggestion
{ stdenv
, fetchGitHub
, libGl
, xorg
, xlibs
, cmake
}:
```

Also, note the usage of `fetchFromGitHub`.

File: pkgs/development/libraries/raylib/default.nix [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+stdenv.mkDerivation {
+  pname = "raylib";
+  version = "2.6.0";
+
+  src = fetchurl {
```
Comment (doronbehar) [inline pkgs/development/libraries/raylib/default.nix:7]: `fetchFromGitHub` here.

**Review by doronbehar — CHANGES_REQUESTED** [review]

**Comment by doronbehar** [comment]
Also, please put everything you do in a single commit with message `raylib: init at 2.6.0`.

**Comment by adamlwgriffiths** [comment]
I didn't realise this PR existed when I created mine (#98029).
FYI, my PR uses the later 3.0.0 version and includes fixes to make audio work.

**Comment by svantepolk** [comment]
Thanks for that, @adamlwgriffiths

**Pull request closed without merging**